### PR TITLE
feat(github): coverage on native evmone test runners

### PR DESCRIPTION
**Pull request by winsvega**
## 🗒️ Description
make coverage script to run on native evmone test runner tools as well 

## ✅ Checklist
- [x] All: Set appropriate labels for the changes.
- [x] All: Considered squashing commits to improve commit history.
- [x] All: Added an entry to [CHANGELOG.md](/ethereum/execution-spec-tests/blob/main/docs/CHANGELOG.md).
- [x] All: Considered updating the online docs in the [./docs/](/ethereum/execution-spec-tests/blob/main/docs/) directory.
- [x] Tests: All converted JSON/YML tests from [ethereum/tests](/ethereum/tests) have been added to [converted-ethereum-tests.txt](/ethereum/execution-spec-tests/blob/main/converted-ethereum-tests.txt).
- [x] Tests: Included the type and version of evm t8n tool used to locally execute test cases:  e.g., ref with commit hash or geth 1.13.1-stable-3f40e65.
- [x] Tests: Ran `mkdocs serve` locally and verified the auto-generated docs for new tests in the [Test Case Reference](https://ethereum.github.io/execution-spec-tests/main/tests/) are correctly formatted.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -10,5 +10,6 @@
 - [ ] All: Added an entry to [CHANGELOG.md](/ethereum/execution-spec-tests/blob/main/docs/CHANGELOG.md).
 - [ ] All: Considered updating the online docs in the [./docs/](/ethereum/execution-spec-tests/blob/main/docs/) directory.
 - [ ] Tests: All converted JSON/YML tests from [ethereum/tests](/ethereum/tests) have been added to [converted-ethereum-tests.txt](/ethereum/execution-spec-tests/blob/main/converted-ethereum-tests.txt).
+- [ ] Tests: All converted JSON/YML tests from [ethereum/tests](/ethereum/tests) have been removed.
```
Comment (marioevz) [inline .github/PULL_REQUEST_TEMPLATE.md:13]: I don't think this check belongs in this repo's PR template. I feel like we should not proactively remove things from `ethereum/tests` until PRs in this repo are merged.

**Review by marioevz — CHANGES_REQUESTED** [review]
LGTM, only maybe remove the PR template change, unless there's a valid reason to include it here.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (winsvega) [reply .github/PULL_REQUEST_TEMPLATE.md:13]: This is so not to forget to attach a pr with removal of the tests from tests repo

**Comment by winsvega** [comment]
I changed the line in PR template so it means not to REMOVE but to at least open removal PR so we merge it one after another. 
1) .py version of the tests merged into pyspecs  ==>  2) old .json tests removed from /tests repo (to avoid dublicates)

**Review by marioevz — APPROVED** [review]
LGTM, thanks!

**Pull request merged**